gmd.py

```
@@ -0,0 +1,331 @@
+    args = parser.parse_args()
+    cfg = load_config()
+
+    if args.cmd == "render":
+        out_dir = args.out_dir or Path(cfg["out_dir_render"]) 
+        out_dir.mkdir(parents=True, exist_ok=True)
+        targets = []
+        if args.input.is_dir():
+            cand = [p for p in sorted(args.input.iterdir()) if p.is_file() and (p.suffix.lower() == ".json" or "." not in p.name)]
+            if args.interactive and shutil.which("fzf"):
+                try:
+                    proc = subprocess.run(["fzf", "--multi"], input="\n".join(str(p) for p in cand).encode("utf-8"), stdout=subprocess.PIPE, stderr=subprocess.PIPE, check=True)
```

Ruff (0.15.9)

[warning] Prefer `capture_output` over sending `stdout` and `stderr` to `PIPE`

Replace with `capture_output` keyword argument

(UP022)